src/client.js

```diff
@@ -65,6 +66,15 @@ function fetchResourcesForProducts(products, client) {
   }, []);
 }
 
+function adaptConfig(config) {
```

Comment:
What's there now is correct. It was `storefrontAccessToken` before which was incorrect.